meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend

```diff
@@ -24,54 +24,44 @@ do_compile[depends] = " \
 
 do_compile:prepend:mx8-nxp-bsp() {
     for target in ${IMXBOOT_TARGETS}; do
```

Comment:
Now we don't use the ${target} inside the loop here. I think we can simplify this part using "if" instead of "for", to avoid duplicate copying.

Reply:
Good catch, it wasn't even needed before, will update.